### ISSUE TITLE
Error while loading configuration: Could not find file '/usr/local/webone/webone.conf'.

### ISSUE THREAD

**Issue by my05** [way5/docker-webone#4]
Hi,

i'm trying to install the latest version and finally succeeded in having it know what webone.conf to use.

But now, when running it i get the following:

WebOne HTTP Proxy Server 0.16.2
https://github.com/atauenis/webone

Using configuration file /usr/local/webone/webone.conf.
Cannot use log file /var/log/webone.log: Access to the path '/var/log/webone.log' is denied..
Configuration load complete.
Creating root SSL Certificate & Private Key for CA...
Unable to create CA Certificate: Access to the path '/ssl.key' is denied..
Supported protocols: HTTP (plain), FTP via Web browser.
Listening for HTTP 1.x on port 8090.

i tried starting it both without sudo and with sudo. No difference.

**Comment by my05** [comment]
I sorted it out. I opened a shell to the container, copying the webone.conf to its "/usr/local/webone" and then running it with

sudo docker run -d -p 8090:8080 -it -v /webone.conf:/usr/local/webone/webone.conf --name webone u306060/webone


mission complete :)

**Comment by way5** [comment]
Hi @my05. There are two ways to install it:
1. Using pre-compiled package, i.e. [deb or rpm package](https://github.com/atauenis/webone/releases/tag/v0.17.0)
2. Compile it from sources yourself. Here you could use [autoinstaller script](https://github.com/way5/webone/blob/master/autoinstall/webone.linux.sh)
And an independent method using [Docker image](https://hub.docker.com/repository/docker/u306060/webone/general).

Also there is newer version available (0.17.0) where a few issues were fixed.

According to what you say, you have chosen compile it yourself.  There are few issues which you can face on that route:
>Access to the path '/var/log/webone.log' is denied.. 

`/var/log` is not accessible to write to, try:

```
touch /var/log/webone.log && chown your_user:your_user_group  /var/log/webone.log
```

> Unable to create CA Certificate: Access to the path '/ssl.key' is denied..

WebOne installation directory is not writable. Your user have no permission to write to `/usr/local/webone`. Try:

```
chown -R your_user:your_user_group /usr/local/webone

or 

chmod -R 775 /usr/local/webone
```

The easiest way to install it is to use method number 1.  You also could use autoinstall method.
